common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -473,11 +467,11 @@ private static CommandResult executeCommand(@SuppressWarnings(WarningType.rawtyp
         final String correlationId = command.getParameters().getCorrelationId();
         if (baseException != null) {
             if (baseException instanceof UserCancelException) {
-                commandResult = new CommandResult(CommandResult.ResultStatus.CANCEL, null,
+                commandResult = CommandResult.ofNull(CommandResult.ResultStatus.CANCEL,
```

Comment:
What does `ofNull` do? 

Also is this a new method that you added in this PR?

Reply:
Yes.  These are static factory methods that allow for the generics to be inferred.